samza-core/src/main/scala/org/apache/samza/config/SystemConfig.scala

```diff
@@ -31,6 +31,11 @@ object SystemConfig {
   val SYSTEM_FACTORY = JavaSystemConfig.SYSTEM_FACTORY_FORMAT
   val CONSUMER_OFFSET_DEFAULT = SYSTEM_PREFIX + "samza.offset.default"
 
+  // If true, automatically delete committed messages from streams whose committed messages are deletable.
```

Comment:
Should be included in the configuration reference, and in javadoc in the config class.

Reply:
Sure. I have updated `configuration-table.html` to include the two newly-added configs. Not sure how to include this in the java doc of the config class. Is there example config whose usage is included in the javadoc in the config class?